README.md

```diff
@@ -1,5 +1,5 @@
 # KFServing
-KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. 
+KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and ONNX as well as inference servers like TensorRT Inference Server and custom container based inference servers. 
```

Comment:
You made this into a run on sentence.

Let's stop updating this list. It says "common frameworks like". Feel free to drop custom containers entirely from the README. It's covered elsewhere.

Reply:
:-)

Reply:
left ONNX in, removed the rest